### PR TITLE
add ckilgannon update brandon

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

**Comment by tmagicturtle** [comment]
This needs to be rebased

**Comment by KrisXV** [comment]
@tmagicturtle it was rebased, it looked worse before this

File: data/mods/ssb/moves.js [hunk 1]
```diff
@@ -991,6 +991,36 @@ let BattleMovedex = {
...
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Black Hole Eclipse', target);
+		},
+		onModifyMove(move, pokemon, target) {
+			if (target.getStat('def', false, true) > target.getStat('spd', false, true)) move.category = 'Physical';
+		},
+		onBasePower(basePower, source, target, move) {
+			if (move.category === 'Physical') {
+				return this.chainModify(2);
```
Comment (HoeenCoder) [inline data/mods/ssb/moves.js:1018]: Really should be modifying the foe's defense before hitting not doubling BP. There should be an event for this.

File: data/mods/ssb/moves.js [hunk 2]
```diff
@@ -991,6 +991,36 @@ let BattleMovedex = {
 		target: "self",
 		type: "Psychic",
 	},
+	// c.kilgannon
+	insidiousassault: {
+		accuracy: 100,
+		basePower: 70,
+		category: "Special",
+		desc: "This move's category becomes physical if the opponent's Defense stat is lower than its Special Defense stat. This move's Base Power is doubled if this move is physical.",
+		shortDesc: "Physical and power doubles if foe's Def < SpD.",
```
Comment (HoeenCoder) [inline data/mods/ssb/moves.js:1000]: Would be wise if possible to mention that defense is halved when this hits.

**Comment by HoeenCoder** [comment]
bump

**Pull request merged**